charts/sqlbackup/Chart.yaml

```diff
@@ -12,7 +12,7 @@ description: A Helm chart to setup psql backups for a desinated time frame
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version
-version: 0.0.1
+version: 0.0.2
 appVersion: 0.0.1
```

Comment:
I wonder if the version bumping to `version` and `appVersion` needs to be formalized.